### ISSUE TITLE
Stop using setAttribute for ARIA

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-player#3378]
Instead of using e.g. `foo.setAttribute('aria-label', 'foo')`, we should be using `foo.ariaLabel = 'foo'`.  The liberal use of setAttribute confuses an internal Google tool that hunts for potential XSS vulnerabilities.

The checks we are having issues with are meant to find an attacker-controllable way to set the 'src' attribute on a `<script>` tag, but in cases where the compiler doesn't know what key we're using or doesn't know the type of the element, it will throw errors and refuse to build our source code.

**Comment by theodab** [comment]
With Alvaro's PR (#3489) merged, the only ARIA property we are assigning with `setAttribute` is now `aria-describedby`, which is set in the demo. Sadly, that attribute does not have a variable equivalent in the `ARIAMixin` interface mixin, so we can't replace it.
So this is as good as we are going to get, I think.
